tests/Platforms/PostgreSQLPlatformTest.php

```diff
@@ -229,7 +229,7 @@ public function testGeneratesSequenceSqlCommands(): void
             $this->platform->getDropSequenceSQL($sequence->getObjectName()->toSQL($this->platform)),
         );
         self::assertEquals(
-            "SELECT NEXTVAL('myseq')",
+            'SELECT NEXTVAL(\'"myseq"\')',
```

Comment:
This change is expected. `nextval()` accepts a `regclass` and coerces a string by resolving it against the catalog, so `myseq` and `"myseq"` will resolve to the same object.